hil/src/flash.rs

```diff
@@ -33,8 +42,8 @@ pub enum FlashVariant {
 impl FlashVariant {
     fn file_name(&self) -> &'static str {
         match self {
-            FlashVariant::Fast => "fastflashcmd.txt",
-            FlashVariant::Regular => "flashcmd.txt",
+            FlashVariant::Fast => "hil-fastflashcmd.txt",
```

Comment:
why the new naming? Keep in mind that `orb-hil` is also a convenience for developers, to run the flashing scripts more automatically. So it should still support regular flashing.